v3/cypress/support/elements/TableTile.ts

```diff
@@ -0,0 +1,126 @@
+export const TableTileElements = {
+    getTableTile() {
+        return cy.get(".codap-component").eq(2);
+    },
+    getCaseTableGrid() {
+        return cy.get("[data-testid=case-table] [role=grid]");
+    },
+    getNumOfAttributes() {
+        return cy.get("[data-testid=case-table] [role=grid]").invoke("attr", "aria-colcount");
+    },
+    getNumOfCases() {
+        return cy.get("[data-testid=case-table] [role=grid]").invoke("attr", "aria-rowcount");
+    },
+    getCollectionTitle() {
+        return this.getTableTile().find("[data-testid=editable-component-title]");
+    },
+    getColumnHeaders() {
+        return cy.get("[data-testid=case-table] [role=columnheader]");
+    },
+    getColumnHeader(index) {
+        return cy.get(".codap-column-header-content").eq(index);
+    },
+    getColumnHeaderTooltip() {
+        return cy.get("[data-testid=case-table-attribute-tooltip]");
+    },
+    openIndexMenuForRow(rowNum) {
+        cy.get("[data-testid=case-table] [role=grid] [role=row][aria-rowindex=\"" + rowNum + "\"] [data-testid=codap-index-content-button]").click();
+    },
+    getIndexMenu() {
+        return cy.get("[data-testid=index-menu-list]");
+    },
+    insertCase() {
+        this.getIndexMenu().should("be.visible");
+        cy.clickMenuItem("Insert Case");
+        cy.wait(500);
+    },
+    insertCases(num_of_cases, location) {
+        this.getIndexMenu().should("be.visible");
+        cy.clickMenuItem("Insert Cases...");
+        cy.get("[data-testid=num-case-input] input").type(num_of_cases)
+        cy.get("[data-testid=add-" + location + "]").click();
+        cy.get("[data-testid=\"Insert Cases-button\"]").contains("Insert Cases").click();
+    },
+    deleteCase() {
+        this.getIndexMenu().should("be.visible");
+        cy.clickMenuItem("Delete Case");
+        cy.wait(500);
+    },
+    getInsertCasesModalHeader() {
+        return cy.get("[data-testid=codap-modal-header]");
+    },
+    closeInsertCasesModal() {
+        cy.get("[data-testid=modal-close-button]").click();
+    },
+    cancelInsertCasesModal() {
+        cy.get("[data-testid=Cancel-button]").click();
+    },
+    getAttributeHeader() {
+        return cy.get("[data-testid^=codap-attribute-button]");
+    },
+    getAttribute(name) {
+        return cy.get("[data-testid^=\"codap-attribute-button " + name + "\"]");
```

Comment:
```suggestion
        return cy.get(`[data-testid^="codap-attribute-button ${name}"]`);
```
Use template strings rather than string concatenation.

Reply:
Done